Cargo.toml

```diff
@@ -211,7 +211,7 @@ nix = { workspace = true, default-features = false, features = [
 nu-test-support = { path = "./crates/nu-test-support", version = "0.91.1" }
 assert_cmd = "2.0"
 dirs-next = { workspace = true }
-divan = "0.1.14"
+divan = { git = "https://github.com/OliverKillane/divan.git", rev = "c54ac74" }
```

Comment:
I thought that git dependencies prevent crates from being published. Not sure if that's the same for [dev-dependencies] though.

Reply:
I am not sure how exactly we publish releases, how would I test this out.

I belive it should not affect, as it should not be built when run with cargo run --release